packages/app-wallet-interface/source/index.html.md

```diff
@@ -275,9 +274,10 @@ Possible response to a `Channel Proposed` event.
 
 ### Errors
 
-| Code | Message           | Meaning                                                          |
-| ---- | ----------------- | ---------------------------------------------------------------- |
-| 1004 | Channel not found | The wallet can't find the channel corresponding to the channelId |
+| Code | Message                  | Meaning                                                          |
+| ---- | ------------------------ | ---------------------------------------------------------------- |
+| 1100 | Channel not found        | The wallet can't find the channel corresponding to the channelId |
+| 1101 | Invalid State Transition | The wallet contains invalid state data                           |
```

Comment:
I think this needs a better description but I couldn't think of anything better.